fn main() -> anyhow::Result<()> {
    console_error_panic_hook::set_once();
    Ok(())
}
